blob/azureblob/example_test.go

```diff
@@ -109,3 +111,56 @@ func Example_openBucketFromURL() {
 	}
 	defer bucket.Close()
 }
+
+func ExampleOpenBucket_usingAADCredentials() {
+	const (
+		// Your Azure Storage Account.
+		accountName = azureblob.AccountName("my-account")
+
+		// Your Azure AAD Service Principal with access to the storage account
+		// https://docs.microsoft.com/en-us/azure/storage/common/storage-auth-aad-app
+		clientID     = "123"
+		clientSecret = "456"
+		tenantID     = "789"
+
+		// The storage container to access.
+		containerName = "my-container"
+	)
+
+	// Get an Oauth2 token for the account for use with Azure Storage
+	ccc := auth.NewClientCredentialsConfig(clientID, clientSecret, tenantID)
+	ccc.Resource = azure.PublicCloud.ResourceIdentifiers.Storage
```

Comment:
This constant doesn't appear to exist in the version we're using. Maybe just replace it with the constant string? (`https://storage.azure.com/`).